setup.py

```diff
@@ -5,14 +5,16 @@
 
 setup(
     name='keras-eval',
-    version='0.0.33',
+    version='0.0.34',
     description='An evaluation abstraction for Keras models.',
     author='Triage Technologies Inc.',
     author_email='ai@triage.com',
     url='https://www.triage.com/',
     packages=find_packages(exclude=['tests', '.cache', '.venv', '.git', 'dist']),
     install_requires=[
-        'Keras>=2.2.0',
+        'numpy==1.14.5',
+        'tensorflow==1.10',
```

Comment:
Recommended to import in `requirements.txt`.

Reply:
what was the reason to move it here?

Reply:
I thought it was maybe because it was not a direct dependency but indirect dependency.